DatadogRUM/Sources/RUMMonitor/Scopes/Utils/RUMViewIdentity.swift

```diff
@@ -85,23 +85,27 @@ internal struct RUMViewIdentity {
     }
 
     /// Returns `true` if a given identifiable indicates the same RUM View as the identifiable managed internally.
-    func equals(_ identifiable: RUMViewIdentifiable?) -> Bool {
-        if let selfObject = object as? RUMViewIdentifiable {
-            return selfObject.equals(identifiable)
-        } else if let selfValue = value as? RUMViewIdentifiable {
-            return selfValue.equals(identifiable)
-        } else {
-            return false
-        }
+    fileprivate func equals(_ identifiable: RUMViewIdentifiable?) -> Bool {
+        return self.identifiable?.equals(identifiable) ?? false
     }
 
     /// Returns `true` if a given identity indicates the same RUM View as the identifiable managed internally.
     func equals(_ identity: RUMViewIdentity?) -> Bool {
         return equals(identity?.identifiable)
     }
 
+    /// Return default path name for the managed identifiable.
+    var defaultViewPath: String {
+        return identifiable?.defaultViewPath ?? ""
+    }
+
+    /// Returns `true` if the managed identifiable is still available.
+    var isIdentifiable: Bool {
+        return identifiable != nil
+    }
```

Comment:
suggestion/ This could be more descriptive. What it does in fact is determining if the underlying source of identity still exists (like VC's reference). I think that calling it "`exists`" would make more sense.